### DISCUSSION TITLE
Registration requires an email, from a third-party?

### DISCUSSION THREAD

**Discussion by Entheologian** [wallabag/wallabag#6423] [Q&A]
According to the documentation: 
> You have to fill the form. Please sure to type a valid email address, we'll send you an activation email.

Does this mean the email addresses are sent to a third-party, rather than through it's own SMTP configured server?

**Comment by nicosomb** [comment]
Hello @Entheologian. 

Thank you for your question. 

About the open source project, if you configure your own SMTP server, you don't send email to a 3rd-party service. 

About wallabag.it, it uses its own SMTP server (no 3rd party service).
